src/core_plugins/metric_vis/public/metric_vis_params.html

```diff
@@ -24,7 +32,7 @@
         kbn-accessible-click
         aria-expanded="{{!!showColorRange}}"
         aria-controls="metricOptionsRanges"
-        aria-label="Toggle range options"
+        aria-label="{{ 'metricVis.public.metricVisParams.ranges.toggleOptionsAriaLabel' | i18n: { defaultMessage: 'Toggle range options' } }}"
```

Comment:
Is there a chance, that `i18n` changes it's result on the fly, while this page is visible? If not we should use bind once syntax here `{{:: }}` to reduce the amount of Angular watches we're having.

Reply:
Labels aren't changed on the fly. Yes, it makes sense to use bind once syntax. 
But there is a tool  that generates json file with ids and default messages. And currently this tool doesn’t support cases with bind once. Since the tool is included in CI job, firstly we need to update our tool, after that we can apply bind once.
@timroes what about if we add bind once syntax as a separate PR once we update our tool?